token-utility-whitepaper.md

```diff
@@ -1,83 +1,77 @@
 
-The diagram below summarises the different types of staking and the distribution of delegated stakes.
+The diagram below summarises the different types of staking and the distribution of delegated stakes.  
+
 
 <p align="left" width="100%">
-    <img width="75%" src="./assets/Delegated-Staking.png">
+    <img width="75%" src="./assets/delegated-staking-diagram-20230601.png">
 </p>
 
-1. Nodes 1 and 2 have the minimum stake value provided by the node operator and they are participants on the Obscuro network. Node 3 does not have the minimum stake therefore it is not a member of the Obscuro network.
+1. Nodes 1 and 2 have the minimum stake quantity provided by the node operator and they are participants on the Obscuro network. Node 3 does not have the minimum stake therefore it is not a member of the Obscuro network.
 2. A delegated stake is provided for node 1 and node 2. Node 2 has the larger delegated stake which has influenced the probability of node 2 winning the rollup lottery.
-3. Node 2 has won the rollup lottery. The reward is taken from the pool of transaction fees and shared proportionally amongst the node operator and delegates for Node 2 in line with the proportion of their staked value.
+3. Node 2 has won the rollup lottery. The reward is given and shared proportionally amongst the node operator and delegates for Node 2 in line with the proportion of quantity of their staked OBX.
 
 There is an occasion where tokens are slashed although this is expected to be very rare. Specifically, operators who use a hacked enclave on their node in an attempt to break the ledger’s integrity will be discovered by the protocol. Their punishment is to have their stake slashed and returned to the Community Reserve.
 
-### Rewards and Fees
-Rewards in Obscuro are an additional incentive to the stake so node operators maintain reliable and accurate nodes thereby contribute to the health of the Obscuro network. During the bootstrapping phase rewards will be limited to the nodes which will be verifying the accuracy of the signed list of transaction hashes generated. After the bootstrapping phase the rewards will be extended to include the nodes producing rollups and publishing to the Layer 1.
+## Rewards
+There are two types of node rewards in Obscuro: a reward for submitting a rollup to the Layer 1, and a reward for making a successful challenge to an inaccurate rollup, for example transactions are missing or in the wrong order. During the bootstrapping phase rewards will be made in ETH and the reward will be limited to those nodes which are verifying the accuracy of the signed list of transaction hashes generated. After the bootstrapping phase the rewards will be made in OBX and the reward model extended to include the nodes producing rollups and publishing to the Layer 1.
```

Comment:
This is a little unclear as there are in total 3 types of rewards for nodes - a reward for submitting a rollup (this will just be the sequencer to begin with), a reward for successful challenge, and a reward from the lottery.